sdk/src/nonce_instruction.rs

```diff
@@ -71,10 +80,22 @@ pub fn create_nonce_account(
             NonceState::size() as u64,
             &id(),
         ),
-        nonce(nonce_pubkey),
+        initialize(nonce_pubkey),
     ]
 }
 
+pub fn initialize(nonce_pubkey: &Pubkey) -> Instruction {
```

Comment:
recommend you not expose initialize() as a raw instruction constructor, instead expose create_account(), a la vote and stake programs

Reply:
Nice eye!  Yep.  I noticed this a little while ago and have it changed locally. Thanks!